app/controllers/sitemaps_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SitemapsController < ApplicationController
+  skip_before_action :http_basic_authenticate # needed for google?
```

Comment:
I don't think this is needed - 2 scenarios I can see

a) its publicly visible, so we want google indexing and this isn't necessary
b) its not publicly visible, so we wont want google to index something thats not accessible?

Reply:
If the entire site is publicly visible, we don't need it. If Lizzie want to start using it now, while the site needs authentication, she would need it?